fixtures/shared/src/run-wrangler-long-lived.ts

```diff
@@ -101,10 +107,25 @@ async function runLongLivedWrangler(
 	}, 50_000);
 
 	async function stop() {
-		const closePromise = events.once(wranglerProcess, "close");
-		wranglerProcess.kill("SIGTERM");
-		const [code] = await closePromise;
-		if (code) throw new Error(`Exited with code ${code}`);
+		return new Promise<void>((resolve) => {
+			assert(
+				wranglerProcess.pid,
+				`Command "${command.join(" ")}" had no process id`
+			);
+			treeKill(wranglerProcess.pid, (e) => {
+				if (e) {
+					console.error(
+						"Failed to kill command: " + command.join(" "),
+						wranglerProcess.pid,
+						e
+					);
+				}
+				// fallthrough to resolve() because either the process is already dead
+				// or don't have permission to kill it or some other reason?
+				// either way, there is nothing we can do and we don't want to fail the test because of this
+				resolve();
+			});
+		});
```

Comment:
This replaces the fixture's runLongLivedWranglers stop implementation with the same implementation from our e2e tests which is more stable (read: actually reliable) than node's child_process.kill('SIGTERM') which sometimes does nothing (especially on windows: always does nothing) causing our tests to hang during cleanup even if they're assertions passed